test/jasmine/tests/finance_test.js

```diff
@@ -963,3 +963,79 @@ describe('finance charts updates:', function() {
         });
     });
 });
+
+describe('finance charts *special* handlers:', function() {
+
+    afterEach(destroyGraphDiv);
+
+    it('`editable: true` handles should work', function(done) {
+
+        function editText(itemNumber, newText) {
+            var textNode = d3.selectAll('text.legendtext')
+                .filter(function(_, i) { return i === itemNumber; }).node();
+            textNode.dispatchEvent(new window.MouseEvent('click'));
+
+            var editNode = d3.select('.plugin-editable.editable').node();
+            editNode.dispatchEvent(new window.FocusEvent('focus'));
+
+            editNode.textContent = newText;
+            editNode.dispatchEvent(new window.FocusEvent('focus'));
```

Comment:
which is used to invoke the [`makeEditable`](https://github.com/plotly/plotly.js/blob/e7f69c986ba11e6672b0ff55bc76389c934d64cb/src/lib/svg_text_utils.js#L436-L535) handlers.